app/assets/javascripts/components/story/StoryDescription.js

```diff
@@ -1,34 +1,39 @@
 import React from 'react';
-import Parser from 'html-react-parser'
-import memoize from 'memoizee';
+import AtWhoInput from 'components/jquery_wrappers/AtWhoInput';
+import DescriptionContent from 'components/description/DescriptionContent'
 
-import StoryLink from 'components/stories/StoryLink';
-
-const editButton = isReadonly =>
-  <input
-    className={!isReadonly && 'edit-description'}
-    disabled={isReadonly}
-    value={I18n.t('edit')}
-    type="button" />
-
-const replaceStoryLink = (domNode, linkedStories) => {
-  if (!domNode.attribs || !domNode.attribs['data-story-id']) return;
-  const id = domNode.attribs['data-story-id'];
-  const story = linkedStories[id];
-  return ( <StoryLink story={story} key={id} /> );
-}
-
-const StoryDescription = ({ description, isReadonly, linkedStories }) => {
-  const isEmpty = (!description || !description.length);
-  description = Parser(description, { replace: domNode =>
-    replaceStoryLink(domNode, linkedStories)
-  });
-
-  return (isEmpty)
-    ? editButton(isReadonly)
-    : <div className='description'>
-        { description }
-      </div>
+const StoryDescription = ({
+  name,
```

Comment:
What do you think about doing this destructuring inside the function body? It'll make it more readable.